scripts/deploy.sh

```diff
@@ -91,7 +91,7 @@ check_create_remote_state $AWS_REGION $STATE_BUCKET_NAME $STATE_DYNAMO_TABLE
 read -rp 'action [init|plan|deploy]: ' ACTION
 case $ACTION in
   init)
-    terraform init
+    terraform init -reconfigure
```

Comment:
```suggestion
    terraform init
init-reconfigure)
    terraform init -reconfigure
```

Could we instead have an option to reconfigure? What would be the benefit to always include reconfiguring on the init? 😯

Reply:
This was a solution to a specific issue I was having on SIT - basically brute-force accepting the current deployed state. Definitely not needed.